scripts/start.js

```diff
@@ -61,6 +68,8 @@ const localhost = '0.0.0.0';
     stats: 'errors-only',
     allowedHosts: appHosts,
     serveIndex: false,
+    hot,
+    clientLogLevel: 'warn',
```

Comment:
Was this debugging, or for better visibility of something specific?

Reply:
This actually hides logging rather than adds it. HMR is super noisy in the console, classic webpack DX.